### DISCUSSION TITLE
CHIRP error "Firmware '' not supported" after flashing v0.22

### DISCUSSION THREAD

**Discussion by therealgglggl** [egzumer/uv-k5-firmware-custom#566] [Q&A] [answered]
Can anyone help with this - I can no longer read/write from/to my uv-k5 with CHIRP (latest version or egzumer 'driver') but the v0.22 seesm to have flashed ok and works fine.

**Comment by AubsUK** [comment]
Have you upgraded to the latest version of CHIRP?

**Reply by therealgglggl** [reply, marked as answer]
Yes, as I said "with CHIRP (latest version or egzumer 'driver') " 
However I just figured it out - its because the programming cable wasnt fully engaged in the radio - not sure why this would be reported as "Firmware '' not supported" ???
At the very least maybe change the message to 'Firmware '' not supported OR check programming cable is fully inserted into the radio'
